MANUAL.txt

```diff
@@ -4179,6 +4179,10 @@ A document may contain multiple metadata blocks.  If two
 metadata blocks attempt to set the same field, the value from
 the second block will be taken.
 
+Each metadata block is an independent YAML document.  This means, for
```

Comment:
This may or may not be relevant to most Pandoc users, but as a programmer using YAML in other contexts I'm used to its features that are explicitly designed to combine other parts of other documents. What this paragraph describes is not a YAML limitation, it is a scoping limitation in Pandoc itself. I would suggest tweaking the wording here a little bit to announce that this is a limitation imposed by Pandoc's YAML handling. Maybe it will save some like myself who might expect more.

```suggestion
Each metadata block is handled internally as an independent YAML document.  This means, for
```

Reply:
I made this change (sorry, I didn't notice the above form until after I'd pushed it). Thanks!